client/src/components/questions/QAModule.jsx

```diff
@@ -2,20 +2,51 @@ import React, {useState, useEffect} from 'react';
 import StyleLib from '../../assets/Stylesheet.jsx';
 import api from '../../lib/api.js';
 import ReviewAndQuestionList from '../../assets/ReviewAndQuestionList.jsx';
+import QuestionList from './QuestionList.jsx';
+import SearchQA from './SearchQA.jsx';
 
-const QAModule = () => {
+const QAModule = ({product}) => {
+  const productID = product.id;
 
+  // React Hooks
+  const [numberOfQuestions, setNumberOfQuestions] = useState(2);
+  const [displayedResults, setDisplayedResults] = useState([]);
+  const [productResults, setProductResults] = useState([]);
+  const [modal, setModal] = useState(false);
+
+  // On Render Effects
+  useEffect(() => {getQuestions()}, []);
+  useEffect(() => {updateDisplayedResults()}, []);
+  useEffect(() => {updateDisplayedResults()}, [numberOfQuestions]);
+
```

Comment:
Nit: similar thing as the previous comment above. I think you can get rid of line 19 and 20 will do everything you need, but I could be wrong.